CHANGELOG.md

```diff
@@ -6,12 +6,17 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ## Master
 
+##### Enhancements
+
+* Replace git-based MasterSource with CDN-based TrunkSource
```

Comment:
nit of the nits you will hate me for it wrap MasterSource and TrunkSource with \`\` (backticks) for formatting.